src/I_GraphicsHandler.h

```diff
@@ -10,6 +10,7 @@ class I_GraphicsHandler {
     virtual void init() = 0;
     virtual void draw(SDL_Texture* texture, SDL_Rect texture_rect, GraphicPriority priority) = 0;
     virtual void draw(SDL_Texture* texture, int x_pos, int y_pos, GraphicPriority priority) = 0;
+    virtual void draw(SDL_Surface* surface, int x_pos, int y_pos, GraphicPriority priority) = 0;
```

Comment:
So with the Terminal, I get the TTF_Font stuff as a surface, and the only way to convert a surface to a texture is if I have access to the renderer. However, we have kept the renderer hidden away in GraphicsHandler so that only graphics handler will do the heavy work. 

That's why I needed graphics handler to accept a surface, and then graphics handler will convert the surface into a texture.